### PR TITLE
Cult Tweaks

### PR DESCRIPTION
**Pull request by Fox-McCloud**
This is meant to technically go along with: https://github.com/ParadiseSS13/Paradise/pull/1942

Just some cult tweaks to address a few things with cult (this game mode really needs to be updated, badly). Plucked from TG's updated cult mode:

- Adds in unholy water
 - Unholy water deals various damage types to non-cultists. Cultists take no damage from it and are more difficult to stun in addition to boosting their movement speed. Cultists and non-cultists alike take brain damage from it.
- Adds in the ability to convert holy water to unholy water with a cult tome (only inside a mob)
 - Simply put if you get holy water in you as a cultist, you can hit yourself with a tome to turn it into unholy water. This allows holy water to deconvert, but actually gives cultists an *advantage* if someone goes around trying to force feed them holy water or holy-water bombs everyone (as they'll just convert it over to unholy water with their tomes).
- Removes Blessing of containers by inserting them into a Bible (this converts water to holy water).
 - Blessing containers that have water isn't removed.
- Revealing runes moved to the Bible itself (which checks if you're a Chaplain).
 - The above two are to help cut down a bit on regular crew doing the "hunting" during cult rounds and instead putting it on who should be: the Chaplain.
- Bible can convert unholy water BACK into holy water

### CODE REVIEW

**Comment by tigercat2000** [comment]
Why remove the blessing of containers in bibles?

**Comment by Fox-McCloud** [comment]
So there's more effort into acquiring holy water than a regular crew member merely getting ahold of a Bible.

**Comment by Alex6511** [comment]
Except every time you need the chaplain he's suddenly spaced himself.

**Comment by Fox-McCloud** [comment]
Then you can go through the process of manually making it in chemistry or ordering holy water flasks from cargo.

Acquiring a bible and sticking in glasses with water in them is 0 effort, requires no time, and can be done on the fly, anywhere.

**Comment by Alex6511** [comment]
I always forget you can make it in chem.

**Comment by Robertha89** [comment]
Yes please, only the Chaplin should be able to bless water to holy water, just like only cultist should be able to bless.. curse? Unbless? Holywater and water to unholy water.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Fox-McCloud** [comment]
forgot about organic/synthetic processing.

unholy water impacts synths, humans, and fuzzybutts alike.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -1064,22 +1064,34 @@ datum
...
-			reaction_obj(var/obj/O, var/volume)
-				var/turf/the_turf = get_turf(O)
-				if(!the_turf)
-					return //No sense trying to start a fire if you don't have a turf to set on fire. --NEO
-				new /obj/effect/decal/cleanable/liquid_fuel(the_turf, volume)
-			reaction_turf(var/turf/T, var/volume)
-				new /obj/effect/decal/cleanable/liquid_fuel(T, volume)
-				return
-			*/
+			..()
```
Comment (TheDZD) [inline code/modules/reagents/Chemistry-Reagents.dm]: This is being called from outside of a proc, is that intentional?

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -1064,22 +1064,34 @@ datum
...
+				process_flags = ORGANIC | SYNTHETIC //ethereal means everything processes it.
+
+				on_mob_life(mob/living/M)
+					M.adjustBrainLoss(3)
+					if(iscultist(M))
+						M.status_flags |= GOTTAGOFAST
+						M.drowsyness = max(M.drowsyness-5, 0)
+						M.AdjustParalysis(-2)
+						M.AdjustStunned(-2)
+						M.AdjustWeakened(-2)
```
Comment (TheDZD) [inline code/modules/reagents/Chemistry-Reagents.dm]: Doesn't this basically just end up being cultist meth?

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 3]
```diff
@@ -1064,22 +1064,34 @@ datum
...
+					else
+						M.adjustToxLoss(2)
+						M.adjustFireLoss(2)
+						M.adjustOxyLoss(2)
+						M.adjustBruteLoss(2)
+					holder.remove_reagent(src.id, 1)
```
Comment (TheDZD) [inline code/modules/reagents/Chemistry-Reagents.dm]: Couldn't this be accomplished by metabolization_rate?

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 2 above)
Comment (Fox-McCloud) [inline code/modules/reagents/Chemistry-Reagents.dm]: Pretty much; that's its intended functionality to punish those who random holywater people.

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/reagents/Chemistry-Reagents.dm]: Nope; needs to be fixed.

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 3 above)
Comment (Fox-McCloud) [inline code/modules/reagents/Chemistry-Reagents.dm]: Technically, yes, but then you have to call parent proc; there's a few differences in how this is handled--I'll be the first to admit, I'm not completely sure why it was done this particular way.

**Pull request merged**